src/pages/markets/components/sub-markets/_digital-options.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import AvailablePlatforms from '../helper/_available-platforms'
+import AvailableOptions from '../helper/_available-options'
+import {
+    Col,
+    ContentWrapper,
+    Descriptions,
+    OptionsRow,
+    StyledText,
+} from '../../static/style/_markets-style'
+import MarketInstruments from '../sections/_market_instruments'
+import type { Options } from '../../static/content/_digital-options'
+import { SectionContainer } from 'components/containers'
+import { Text } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+
+type DigitalOptionsProps = {
+    market_name: string
+    market_type?: string
+    options_list: Options
+}
+
+const DigitalOptions = ({ market_name, market_type, options_list }: DigitalOptionsProps) => {
+    const what_to_show = market_name === 'forex' && market_type === 'basket-indices'
+
+    return (
+        <SectionContainer padding="4rem 0 8rem">
+            <ContentWrapper>
+                <Descriptions>
+                    <StyledText font_size="16px" align="center">
+                        <Localize translate_text="Options trading allows for fixed payouts from correctly predicting market movements, without needing to buy an underlying asset." />
+                    </StyledText>
+                    {what_to_show ? (
+                        <AvailablePlatforms dmt5 derivx />
+                    ) : (
+                        <AvailablePlatforms dtrader dbot smarttrader tablet_direction="column" />
+                    )}
+                </Descriptions>
+                <StyledText font_size="16px" weight="bold" mt="4.0rem">
+                    <Localize
+                        translate_text="Option trades available on {{market_name}}"
+                        values={{ market_name }}
+                    />
+                </StyledText>
+                <Descriptions margin_top="24px">
+                    {options_list.options.map((option, index) => (
+                        <OptionsRow wrap={option[0].wrap} key={index} is_first_child={!index}>
+                            {option.map((content, idx) => (
+                                <Col key={idx}>
+                                    <AvailableOptions content={content.text} {...content} />
+                                </Col>
+                            ))}
+                        </OptionsRow>
+                    ))}
+                    {/* no content for this component inside data  */}
+                    {/* <Show.Eu>
+                        {options_list.eu_content?.map((text, index) => (
+                            <Text key={index} mt="1.6rem" color="grey-5" size="var(--text-size-xs)">
+                                {text}
+                            </Text>
+                        ))}
+                    </Show.Eu>  */}
+                </Descriptions>
+                <Text weight="bold" mt="4.0rem">
```

Comment:
Since the `Text` component is deprecated and we use `Header` instead also this is a new file, it would be better to not use it so later we won’t need to come back to this file and refactor it. 🙏

Reply:
@mitra-fs  Hi. Please, could you make small video, I can't reproduce  no scroll on this carousel?  
Also I'll  replace Text there

Reply:
Sure, here is the video attached 🙏 

https://user-images.githubusercontent.com/64970259/158186158-57607387-d244-442a-a113-833154f0aada.mov